contracts/MintAndBurnAuthorityV2.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.4.24;
+
+contract MintAndBurnAuthority {
```

Comment:
This is more secure version but with less ease of use. I prefer to use this version for safety concern.